### ISSUE TITLE
traceback "AttributeError: 'NoneType' object has no attribute 'qname'"

### ISSUE THREAD

**Issue by RemiCardona** [PyCQA/pylint#2316]
```
pylint --rcfile=`dev-config-path .pylintrc` --jobs=1 $modules
************* Module statio.telemetry_raw_service
statio/telemetry_raw_service.py:147: [E1133(not-an-iterable), funcs.slow] Non-iterable value get_telemetry(fleet_settings, fast=False, loop=loop) is used in an iterating context
statio/telemetry_raw_service.py:159: [E1133(not-an-iterable), funcs.fast] Non-iterable value get_telemetry(fleet_settings, fast=True, loop=loop) is used in an iterating context
Traceback (most recent call last):
  File "/home/jenkins/venv/5/bin/pylint", line 11, in <module>
    sys.exit(run_pylint())
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/__init__.py", line 19, in run_pylint
    Run(sys.argv[1:])
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/lint.py", line 1394, in __init__
    linter.check(args)
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/lint.py", line 801, in check
    self._do_check(files_or_modules)
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/lint.py", line 938, in _do_check
    self.check_astroid_module(ast_node, walker, rawcheckers, tokencheckers)
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/lint.py", line 1018, in check_astroid_module
    walker.walk(ast_node)
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/utils.py", line 1159, in walk
    self.walk(child)
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/utils.py", line 1159, in walk
    self.walk(child)
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/utils.py", line 1159, in walk
    self.walk(child)
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/utils.py", line 1159, in walk
    self.walk(child)
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/utils.py", line 1159, in walk
    self.walk(child)
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/utils.py", line 1156, in walk
    cb(astroid)
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/checkers/refactoring.py", line 470, in visit_call
    self._check_raising_stopiteration_in_generator_next_call(node)
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/checkers/refactoring.py", line 502, in _check_raising_stopiteration_in_generator_next_call
    and not _looks_like_infinite_iterator(node.args[0])):
  File "/home/jenkins/venv/5/lib/python3.4/site-packages/pylint/checkers/refactoring.py", line 485, in _looks_like_infinite_iterator
    return inferred.qname() in KNOWN_INFINITE_ITERATORS
AttributeError: 'NoneType' object has no attribute 'qname'
```

### Steps to reproduce
1. Not sure yet, I'll "bisect" my project files to figure out which bit of code breaks pylint

### pylint --version output
```
pylint 2.0.0
astroid 2.0.0.dev4
Python 3.4.2 (default, Oct  8 2014, 10:45:20) 
[GCC 4.9.1]
```

**Comment by PCManticore** [comment]
Thanks for the report! It's now fixed in `master`. We'll most likely do a release next week with fixes related to 2.0.

**Comment by RemiCardona** [comment]
I found the file that was breaking pylint. Anyway pylint from git master no longer fails.

May I request a release sooner rather than later?

Either way, thanks for fixing this!
